.github/workflows/unit_tests.yml

```
@@ -62,7 +62,7 @@ jobs:
         run: |
           testenv\Scripts\Activate.ps1
           pip install pytest
-          pytest -v pyaedt/desktop.py -p no:faulthandler
+          pytest -v pyaedt/desktop.py pyaedt/icepak.py -p no:faulthandler
```

```suggestion
          pytest -v pyaedt/desktop.py pyaedt/icepak.py
```
We can remove the option `no:faulthandler` here as it's in `pytest.ini`, correct?

---

@akaszynski yes, that's correct.